lib/core-generators/new/templates/views/404-caviar.ejs

```diff
@@ -1,10 +1,10 @@
 <div id="404">
-  <div class="container text-center">
+  <div class="container-md text-center pt-5 pb-5s">
```

Comment:
the `container-md`, `container-xl`, etc. bootstrap classes aren't supported in the version of bootstrap we've currently got bundled. I think it makes sense to look into switching to the latest version, but we'd probably want to do that in a separate PR and possibly do some other cleanup along with it. 

for now, let's just give the container a `max-width` of whatever that `.container-md()` mixin uses, along with the `mx-auto` class for centering